pom.xml

```diff
@@ -148,6 +149,23 @@ THE SOFTWARE.
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>${junit.jupiter.version}</version>
+    </dependency>
```

Comment:
Hope this alone does not cause problems like [SUREFIRE-1911](https://issues.apache.org/jira/browse/SUREFIRE-1911).

Reply:
Oh dear! That would be unfortunate :-(

Reply:
I am afraid that acc. to `git bisect` on https://github.com/jenkinsci/plugin-pom/pull/416 it does: with this patch, _no_ JUnit 4 tests are run.

Reply:
⇒ #317

Reply:
According to https://github.com/jenkins-infra/pipeline-library/pull/272, `junit-vintage-engine` shoudl have been added with scope `compile` instead of `test`.